manifests/n/NetSarangComputer,Inc/Xshell8/8.0.0069/NetSarangComputer,Inc.Xshell8.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.9.14.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: NetSarangComputer,Inc.Xshell8

```

```suggestion
PackageIdentifier: NetSarangComputer.Xshell8
```
A publisher folder already exists for `NetSarangComputer`, this manifest should be put under that same publisher